Explorer/Assets/DCL/Multiplayer/Movement/Systems/MultiplayerMovementDebug.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Arch.Core;
+using DCL.AssetsProvision;
+using DCL.Character.Components;
+using DCL.DebugUtilities;
+using DCL.Multiplayer.Movement.Settings;
+using DCL.Multiplayer.Profiles.Entities;
+using DCL.Multiplayer.Profiles.RemoteProfiles;
+using DCL.Profiles;
+using System;
+using UnityEngine;
+using Utility;
+
+namespace DCL.Multiplayer.Movement.Systems
+{
+    public class MultiplayerMovementDebug : IDisposable
+    {
+        private readonly RemoteEntities remoteEntities;
+        private readonly ExposedTransform playerTransform;
+        private readonly ProvidedAsset<MultiplayerMovementSettings> settings;
+
+        private Entity? selfReplicaEntity;
+
+        public MultiplayerMovementDebug(World world, IDebugContainerBuilder debugBuilder, RemoteEntities remoteEntities, ExposedTransform playerTransform, ProvidedAsset<MultiplayerMovementSettings> settings)
+        {
+            this.remoteEntities = remoteEntities;
+            this.playerTransform = playerTransform;
+            this.settings = settings;
+
+            debugBuilder.TryAddWidget("Multiplayer Movement")
+                       ?.AddSingleButton("Instantiate Self-Replica", () => InstantiateSelfReplica(world))
```

Comment:
Perhaps replace `?` with `!` or add two `?`. If its null it will break on the 2nd call of `AddSingleButton`, i'm right?

Reply:
I think if it is null, it will not execute further. I don't remember why in other similar code we put `?`, but I will try to change it to `!`